### ISSUE TITLE
Philips Hue bluetooth integration without bridge

### ISSUE THREAD

**Issue by Aidanvii7** [Jairzinno/hue_dart#3]
Does this work with bluetooth enabled Philips Hue bulbs that don't require a bridge?

Side note: I noticed the Philips Hue BT app is built with Flutter

**Comment by matzesoft** [comment]
I do not think it is currently possible to control your BT Hue Lights with this libary.
So better move this to a pull request.

But when Philips managed it to develop their app with Flutter - there must be a solution to get it all working. I think that calls for a big coding night ;).

**Comment by guyluz11** [comment]
Joining the question
